vision/interface.py

```diff
@@ -23,3 +23,18 @@ def __iter__(self):
     #updates rects list
```

Comment:
Lets switch this comment to a docstring and also put it right under the definition
ie
```python
def update(self, bounding_boxes):
    """
    description

    Parameters
    ------------
    bounding_boxes: type
         description
```